<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>选择文件：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textSelectPath"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnSelectFile">
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabDesc">
       <attribute name="title">
        <string>简述</string>
       </attribute>
       <widget class="QPlainTextEdit" name="textPEAll">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>761</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabDos">
       <attribute name="title">
        <string>DOS文件头</string>
       </attribute>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>761</width>
          <height>581</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>10</y>
           <width>90</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>【02h MZ头】</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>540</y>
           <width>161</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>【04h PE文件头地址】*</string>
         </property>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>101</width>
           <height>551</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>e_magic</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>e_cblp</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>e_cp</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>e_crlc</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>e_cparhdr</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>e_minalloc</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>e_maxalloc</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>e_ss</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>e_sp</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>e_csum</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>e_ip</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>e_cs</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>e_lfarlc</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>e_ovno</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="14" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>e_res</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="15" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>e_oemid</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="16" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>e_oeminfo</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="17" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>e_res2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="18" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>e_lfanew</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>10</y>
           <width>321</width>
           <height>551</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLineEdit" name="e_magic"/>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="e_cblp"/>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="e_cp"/>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="e_crlc"/>
          </item>
          <item row="4" column="0">
           <widget class="QLineEdit" name="e_cparhdr"/>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="e_minalloc"/>
          </item>
          <item row="6" column="0">
           <widget class="QLineEdit" name="e_maxalloc"/>
          </item>
          <item row="7" column="0">
           <widget class="QLineEdit" name="e_ss"/>
          </item>
          <item row="8" column="0">
           <widget class="QLineEdit" name="e_sp"/>
          </item>
          <item row="9" column="0">
           <widget class="QLineEdit" name="e_csum"/>
          </item>
          <item row="10" column="0">
           <widget class="QLineEdit" name="e_ip"/>
          </item>
          <item row="11" column="0">
           <widget class="QLineEdit" name="e_cs"/>
          </item>
          <item row="12" column="0">
           <widget class="QLineEdit" name="e_lfarlc"/>
          </item>
          <item row="13" column="0">
           <widget class="QLineEdit" name="e_ovno"/>
          </item>
          <item row="14" column="0">
           <widget class="QLineEdit" name="e_res"/>
          </item>
          <item row="15" column="0">
           <widget class="QLineEdit" name="e_oemid"/>
          </item>
          <item row="16" column="0">
           <widget class="QLineEdit" name="e_oeminfo"/>
          </item>
          <item row="17" column="0">
           <widget class="QLineEdit" name="e_res2"/>
          </item>
          <item row="18" column="0">
           <widget class="QLineEdit" name="e_lfanew"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
